webfont/make_woff.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python3
+import sys
+from sys import exit, argv
+from pathlib import Path
+import json
+import logging
+
+
+def error(*args):
+    print(*args, file=sys.stderr)
+
+
+if sys.version_info[0:2] < (3, 6):
+    error("This script requires Python version 3.6")
+    exit(1)
+
+
+try:
+    import fontTools.ttLib
+    import fontTools.subset
+    import fontTools.merge
+    import fontTools.misc.loggingTools
+except ImportError:
+    error("Please install the fontTools python module.")
```

Comment:
it feels like we should be able to invoke pip here.
```
accepted = ask("FontTools dependency is missing, would you like to install them now? [Y/n]");
if accepted is True:
    runPipInstalls()
else:
    error("Dependencies not installed, exiting...")
    exit(1)
```

Reply:
There are just so many possibilities. Do you want it to be installed with the distribution's package manager or pip? Install globally (which also needs root permission probably) or only for the current user or maybe in a virtual environment? But I can make something if it's really necessary.

Reply:
Offering the choice, only makes it more usable. We can add in the option to use `pip` (which is the defacto manager these days) and if someone says "no" then they get exactly as much functionality as if we just immediately exit. So worst case: things stay the same. Best case: the script takes care of it and just continues. 

Let's add in a pip install (and if it complains about permissions, we can make it say "could not install dependencies, please install the following packages manually:" and then a list of the dependencies necessary)